### PR TITLE
updated index page and download text

### PR DESCRIPTION
**Pull request by nylim11**

### CODE REVIEW

File: src/pages/index.hbs [hunk 1]
```diff
@@ -55,24 +55,34 @@ menuItems:
       <section id="main" class="container">
 
         <section class="box special">
+          <div style="position: relative; float: left">
```
Comment (chrislim) [inline src/pages/index.hbs]: do you need position:relative?
Also have you tested on other devices/screen sizes?

File: src/pages/index.hbs [hunk 2]
```diff
@@ -55,24 +55,34 @@ menuItems:
...
+            <script src="//fast.wistia.com/embed/medias/h0rqwj77w6.jsonp" async></script>
+            <script src="//fast.wistia.com/assets/external/E-v1.js" async></script>
+            <div class="wistia_embed wistia_async_h0rqwj77w6" style="height:300px;width:169px">&nbsp;</div>
+          </div>
+          <!--<span><img src="images/iphone/Screenshot1-croppedindex.png" style="float:left" alt="Ceaseless App Screenshot"></span>-->
```
Comment (chrislim) [inline src/pages/index.hbs]: we probably don't need this...

File: src/pages/index.hbs [hunk 3]
```diff
@@ -55,24 +55,34 @@ menuItems:
...
           <header class="major">
             <h2>Prayer shouldn't feel like a to-do list.</h2>
           </header>
           <p>
-            Ceaseless helps you experience the joy of prayer by showing you a Scripture and three people to pray for each day.
-            You can add notes from your conversations with them and use it as a prayer journal.
+            Ceaseless is an app that helps you experience the joy of prayer by showing you a Scripture and three people to pray for each day.
+            You can add notes from your conversations with them and use it as a private prayer journal.
+            Send an encouraging message letting them know you prayed for them.
+            <br/><br/><b>It's a simple and enjoyable way to pray every day!</b>
```
Comment (chrislim) [inline src/pages/index.hbs]: Should this be brs or a separate p?

**Review by chrislim — APPROVED** [review]
minor changes and you can ship it.

File: src/pages/index.hbs (hunk 2 above)
Comment (nylim11) [reply src/pages/index.hbs]: removed

File: src/pages/index.hbs (hunk 3 above)
Comment (nylim11) [reply src/pages/index.hbs]: fixed

**Review by chrislim — APPROVED** [review]

**Pull request merged**